File: flows/disaster/scriptlets/+scriptlet#1.py
from json import JSONDecodeError
import typing
#region generated meta
from oocana import Context, LLMModelOptions
class Inputs(typing.TypedDict):
    input: LLMModelOptions
class Outputs(typing.TypedDict):
    output: typing.Any
#endregion

def main(params: Inputs, context: Context) -> Outputs:
    # your code

    return { "output": "output_value" }

File: flows/disaster/scriptlets/+scriptlet#1.js
//#region generated meta
/**
 * @import { Context, LLMModelOptions } from "@oomol/types/oocana";
 * @typedef {{
 *   input: LLMModelOptions;
 * }} Inputs;
 * @typedef {{
 *   output: any;
 * }} Outputs;
 */
//#endregion

import { subscribe } from "diagnostics_channel";

/**
 * @param {Inputs} params
 * @param {Context<Inputs, Outputs>} context
 * @returns {Promise<Outputs>}
 */
export default async function (params, context) {

    // your code
    return { output: "output_value" };
}
